vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -198,36 +209,37 @@ fire_and_forget WinRTWebSocketResource::PerformWrite()
 
     if (isBinary)
     {
-      m_socket.Control().MessageType(SocketMessageType::Binary);
+      self->m_socket.Control().MessageType(SocketMessageType::Binary);
 
       auto buffer = CryptographicBuffer::DecodeFromBase64String(Utf8ToUtf16(std::move(message)));
       length = buffer.Length();
-      m_writer.WriteBuffer(buffer);
+      self->m_writer.WriteBuffer(buffer);
     }
     else
     {
-      m_socket.Control().MessageType(SocketMessageType::Utf8);
+      self->m_socket.Control().MessageType(SocketMessageType::Utf8);
 
       //TODO: Use char_t instead of uint8_t?
       length = message.size();
       winrt::array_view<const uint8_t> arr(
         CheckedReinterpretCast<const uint8_t*>(message.c_str()),
         CheckedReinterpretCast<const uint8_t*>(message.c_str()) + message.length());
-      m_writer.WriteBytes(arr);
+      self->m_writer.WriteBytes(arr);
     }
 
-    co_await m_writer.StoreAsync();
+    auto operation = m_writer.StoreAsync();
```

Comment:
nit: `self->m_writer` for consistency. Though accessing raw member variables is safe here, so long as you have the strong ptr in scope. I.e. you can remove the `self->` entirely if you wanted to.

Reply:
Noted.
I'll just add the missing `self->` to make the change simpler.
Will remove in a subsequent change.

Reply:
Another leftover from the `weak_ptr` attempt.
There is no need to store the action/operation in a variable anymore.